### DISCUSSION TITLE
Hosting a saved webpage?

### DISCUSSION THREAD

**Discussion by ThisIsJeron** [community/community#21670] [Other Feature Feedback, Questions, & Ideas]
<p>I’m trying to host a saved webpage (ie. chrome webpage complete or safari’s webarchive) using github pages for a school project. While the html file opens on github pages just fine, it seems that the saved webpage includes some php and js for styling (with css, I believe all of them are used), and so styling is not visible. I know php is not usable on GitHub pages (static pages only), but the webpage opens without a problem locally, making me wonder if it’s an issue/incompatibility with GitHub pages. <a href="https://github.com/ThisIsJeron/anth230wiki" rel="nofollow noopener">Here</a> is the source code </p>
<p>thanks in advance!</p>
<p>e: also <a href="https://thisisjeron.github.io/anth230wiki/" rel="nofollow noopener">here</a> is what it looks like right now, styleless </p>

**Comment by sharegrams** [comment]
<p>Hello Jeron,</p>
<p>I’m looking at the repo you linked, is this the one you’ve been using for Github Pages? It will not work because the name of the repo must be <a href="http://USERNAME.github.io" rel="nofollow noopener">USERNAME.github.io</a></p>
<p>In order for your domain to work</p>

**Comment by ThisIsJeron** [comment]
<p>Thanks for the reply, that is the repo I am using, domain is not the issue and the site is accessible. The issue is that it is not using the style files that are in that repo </p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/thisisjeron">@thisisjeron</a>,</p>
<p>Thanks for being here! You have this line on your index.html</p>
<pre><code>link rel="stylesheet" href="./Ward Goodenough - Wikipedia_files/load.php
</code></pre>
<p>Which indicate to load a PHP file. GitHub pages hosting service is intended to host static web pages for GitHub users.”, PHP is not static therefore it will not execute. </p>
<p>I hope this helps!</p>
<p>Andrea</p>
